docker-compose.yml

```diff
@@ -21,10 +22,11 @@ services:
             context: .
             dockerfile: ./netpalm_pinned_worker_dockerfile
         environment:
-            - NET_TEXTFSM=/usr/local/lib/python3.8/site-packages/ntc_templates/templates/
-            - NETPALM_CONFIG=config.json
+          - NET_TEXTFSM=/usr/local/lib/python3.8/site-packages/ntc_templates/templates/
+          - NETPALM_CONFIG=/code/config.json
+          - NETPALM_LOG_CONFIG_FILENAME=/code/log-config.yml
         depends_on:
-            - netpalm-controller
```

Comment:
Any particular reason for changing this?

Reply:
faster turnup, etc.  The workers can run (with nothing to do) without the controller.  But they could break if redis isn't ready and waiting by the time they try to connect.  So workers and controllers all depend on redis, who depends on no one.